src/fal/run_scripts.py

```diff
@@ -86,6 +86,14 @@ def _prepare_exec_script(script: FalScript, project: FalProject) -> bool:
     return success
 
 
+def handle_run_results(scripts: List[FalScript], results: List[bool]):
```

Comment:
Why don't we incorporate some of this into `run_scripts`? Maybe we can do mapping/filtering there and just raise an Error in a runner.

Reply:
Yes, you are right. What if we also raise in the run_scripts for now? Just make it do everything about a script and later we change if we need to. What do you think?

Reply:
Wouldn't this stop the entire fal run? Ideally we want to finish running everything we can and then raise an error.

Reply:
I think we are OK like this for now, and we eventually will move to printing dbt-style logging like

```
Concurrency: 4 threads (target='dev')
1 of 2 START global script scripts/a.py
2 of 2 START model_a script scripts/a.py
2 of 2 OK model_a script scripts/a.py
1 of 2 OK global script scripts/a.py
```

Or something like that